callisto_core/accounts/auth.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+from hashlib import sha256
+
+import bcrypt
+
+from django.conf import settings
+from django.contrib.auth.hashers import check_password
+from django.contrib.auth.models import User
+
+from callisto_core.accounts.models import Account
+
+
+def index(src, length=8):
+    """Computes an index suitable for fast lookups on a user record."""
+    base = f"{src}//{settings.INDEXING_KEY}"
```

Comment:
This should also be configured in `callisto_core/utils/settings.py` with `os.getenv()` (examples of such are in the file)

Reply:
To clarify, by "this" i mean "settings.INDEXING_KEY"